odoo/addons/base/models/res_company.py

```diff
@@ -315,7 +315,7 @@ def create(self, vals_list):
         companies.currency_id.sudo().filtered(lambda c: not c.active).active = True
 
         companies_needs_l10n = companies.filtered('country_id')
-        if companies_needs_l10n:
+        if companies_needs_l10n and not self.env.context.get('install_mode'):
```

Comment:
If this solves the issue, can you move this check inside the ionstall_l10n_modules function?